src/layers/drop.jl

```diff
@@ -0,0 +1,41 @@
+using Flux:Zygote
```

Comment:
About hard-coding `dims=4`, see my comment about layernorm and figuring out a way to support relative dims from the end upstream in Flux. For now I think it's fine to only work with 2D images.

Reply:
dims=4 is a really nifty trick....I missed that when I was searching because I was looking specifically for stochastic depth layers but that is exactly what's happening here so I think it's perfect